### DISCUSSION TITLE
Searchable syntax for code blocks...

### DISCUSSION THREAD

**Discussion by k1rb** [outline/outline#6792] [Ideas]
Can we somehow bring back the ability to quickly select your code block syntax without scrolling through the list and clicking?

In previous releases you could click the drop down and search or skip forward by typing the language, then hit enter to select. I asked chatgpt what this would be called and it says:

_"The feature you're describing, where you can type to quickly navigate or jump to a specific item in a graphical user interface (GUI) list, is commonly referred to as "type-ahead find," "incremental search," or "quick search." This functionality allows users to efficiently locate an item in a long list or dropdown menu by typing the first few letters of the item's name, thereby skipping directly to it without having to scroll through the entire list. This feature enhances user experience by making data retrieval within GUIs faster and more intuitive."_

This sounds right to me.

Thanks!

**Comment by tommoor** [comment]
Thanks for raising, just fixed here 2e427a1e8
